test/evaluation/test_map.py

```diff
@@ -0,0 +1,91 @@
+from collections import defaultdict
+from typing import Dict, Sequence
+
+import numpy as np
+from numpy.testing import assert_allclose
+from pytest import mark, skip
+
+from rsdiv.evaluation import RankingMetrics
+
+
+class TestMAP:
+    rng = np.random.default_rng(15213)
```

Comment:
The practice is to choose random state as either `0` or `42`

Reply:
15213 is actually CMU's zip code. 😂